### ISSUE TITLE
Magazine doesn't load correctly in Creative Mode

### ISSUE THREAD

**Issue by BottleCapBeaver** [Cubed-Development/Modern-Warfare-Cubed#301]
## Describe the bug

When I load any gun in creative mode it loads either an error model or the maximum capacity magazine instead of the magazine I have in my inventory. This does not happen in survival, where it instead loads correctly.

For example, if I am holding a M4A1 and have a 30rnd PMAG in my inventory, it loads instead a bugged model of some 100 round magazine.

I have no other mods (except for Red Core) installed

This happens with many other guns too like the Glock 19

It would make sense for the M4A1 to just load the PMAG.

- [x] I reproduced this issue with as few other mods as possible installed.
- [ ] I am unable to reproduce this issue consistently.

## Versions

- Modern-Warfare-Cubed-0.1-Dev-14
- Forge: [e.g. 1.12.2 - 14.23.5.2860]

**Comment by le-papudinho** [comment]
On creative mode you always load the highest capacity mag, in order to reload from your inventory, you have to reload while sneaking (Defalut Shift). And about the bugged texture drum mag, it will be fixed on the next update

**Comment by BottleCapBeaver** [comment]
> On creative mode you always load the highest capacity mag, in order to reload from your inventory, you have to reload while sneaking (Defalut Shift). And about the bugged texture drum mag, it will be fixed on the next update

Oh I see, thanks for explaining! 
Though is it possible to make it always load the magazine in inventory by default like it used to be?

**Comment by hfgm1145** [comment]
m9a1 .44 can use 65rnd 9x19mm drum mag...it's incredible

**Comment by thelegendaryjohn** [comment]
> > On creative mode you always load the highest capacity mag, in order to reload from your inventory, you have to reload while sneaking (Defalut Shift). And about the bugged texture drum mag, it will be fixed on the next update
> 
> Oh I see, thanks for explaining! 
> Though is it possible to make it always load the magazine in inventory by default like it used to be?

I mean, you can always petition for a feature that does precisely that

**Comment by Desoroxxx** [comment]
> > On creative mode you always load the highest capacity mag, in order to reload from your inventory, you have to reload while sneaking (Defalut Shift). And about the bugged texture drum mag, it will be fixed on the next update
> 
> Oh I see, thanks for explaining! Though is it possible to make it always load the magazine in inventory by default like it used to be?

No, Minecraft does not and never worked like that, the way it was done before was a hack that maybe is nice for people playing in creative, but was a nightmare for addon authors and plugin authors making it impossible for them to create an empty magazine like any other Minecraft item.
